dev/com.ibm.ws.security.saml.sso_fat.common/SAMLCommon.gradle

```diff
@@ -0,0 +1,549 @@
+      from new File(project(':com.ibm.ws.security.saml.sso_fat.common').projectDir, 'publish/files/serversettings')
+      into new File(autoFvtDir, "publish/servers/${serverName}/imports")
+      include '**misc.xml'
+    }
+    copy {
+      from new File(project(':fattest.simplicity').projectDir, 'autoFVT-defaults')
+      into new File(autoFvtDir, "publish/servers/${serverName}/imports")
+      include "**fatTestCommon.xml"
+    }
+    // copyCommonServerXmlRegistry
+    copy {
+      from new File(project(':com.ibm.ws.security.saml.sso_fat.common').projectDir, 'publish/files/serversettings')
+      into new File(autoFvtDir, "publish/servers/${serverName}/imports")
+      include '**egistry*.xml'
+    }
+    // copyCommonServerXmlSSL
+    copy {
+      from new File(project(':com.ibm.ws.security.saml.sso_fat.common').projectDir, 'publish/files/serversettings')
+      into new File(autoFvtDir, "publish/servers/${serverName}/imports")
+      include '**ssl*.xml', '**SSL*.xml'
+    }
+    // copyCommonServerXmlProvider
+    copy {
+      from new File(project(':com.ibm.ws.security.saml.sso_fat.common').projectDir, 'publish/files/serversettings')
+      into new File(autoFvtDir, "publish/servers/${serverName}/imports")
+      include '**rovider*.xml', '**.base'
+    }
+    copy {
+      from new File(project(':com.ibm.ws.security.saml.sso_fat.common').projectDir, 'publish/files/serversettings/SAMLServerFiles')
+      into new File(autoFvtDir, "publish/servers/${serverName}/imports")
+      include '**/**'
+    }
+    // copyCommonServerXmlApplications
+    copy {
+      from new File(project(':com.ibm.ws.security.saml.sso_fat.common').projectDir, 'publish/files/serversettings')
+      into new File(autoFvtDir, "publish/servers/${serverName}/imports")
+      include '**Application*.xml'
+    }
+    // copyServerXmlParts
+    copy {
+      from new File(projectDir, 'publish/files/serversettings')
+      into new File(autoFvtDir, "publish/servers/${serverName}/imports")
+      include '**.xml'
+      include '**.base'
+    }
+    copy {
+      from new File(projectDir, 'publish/files/serversettings/SAMLServerFiles')
+      into new File(autoFvtDir, "publish/servers/${serverName}/imports")
+      include '**/**'
+    }
+  
+    // Copy files to LibertyFATTestFiles. This occurred in WS-CD-Open automatically b/c the files
+    // were compiled in the source directories instead of the build directories.
+    copy {
+      from new File(autoFvtDir, "publish/servers/${serverName}/imports")
+      into new File(autoFvtDir, 'lib/LibertyFATTestFiles/serversettings/SAMLServerFiles')
+      include '**/*.orig'
+    }
+    if (project.findProperty('helloworldWar')) {
+      copy {
+        from new File(project(':com.ibm.ws.security.saml.sso_fat.jaxrs').buildDir, 'test-application/helloworld.war')
+        into new File(autoFvtDir, "publish/servers/${serverName}/test-apps")
+      }
+    }
+    if (project.findProperty('httpServletReuqestAppWar')) {
```

Comment:
Just FYI to avoid possible confusion later - "request" is misspelled. Everywhere uses the same name, so everything will work for now, but just wanted to note it in case someone tries to manually type it elsewhere.

Reply:
never said I could type or spell :)
I can update that when I update the github test-categories